akka-actor-typed-tests/src/test/scala/akka/actor/typed/LocalActorRefProviderLogMessagesSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
+
+import akka.actor.testkit.typed.scaladsl.{ LoggingTestKit, ScalaTestWithActorTestKit }
+import akka.actor.typed.internal.adapter.ActorSystemAdapter
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object LocalActorRefProviderLogMessagesSpec {
+  val config = """
+    akka {
+      loglevel = DEBUG # test verifies debug
+      log-dead-letters = on
+      actor {
+        debug.unhandled = on
+      }
+    }
+  """
+}
+
+class LocalActorRefProviderLogMessagesSpec
+    extends ScalaTestWithActorTestKit(LocalActorRefProviderLogMessagesSpec.config)
+    with AnyWordSpecLike {
+
+  "An LocalActorRefProvider" must {
+
+    "logs on dedicated 'serialization' logger when an ActorRef can't be deserialized" in {
```

Comment:
Since we are adding logging tests, we may want add test for all four deserialization cases.

Reply:
not very important in my opinion, rather trivial code

Reply:
well, while doing that I discover something that might be a bug. Will commit soon...